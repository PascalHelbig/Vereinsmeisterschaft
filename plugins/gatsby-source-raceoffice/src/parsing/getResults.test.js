const fs = require('fs');
const path = require('path');
const getResults = require('./getResults');

it('matches snapshot', () => {
  const fileName = path.join(__dirname, '/__mocks__/results.html');
  const html = fs.readFileSync(fileName, 'utf-8');
  const results = getResults(html);
  expect(results).toMatchSnapshot();
});
