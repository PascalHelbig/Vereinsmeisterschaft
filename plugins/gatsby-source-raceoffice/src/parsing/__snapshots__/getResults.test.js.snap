// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  Object {
    "crew": "Jakob Sell",
    "crewClub": "SCZ",
    "crewClubCode": "B.122",
    "helm": "Pascal Helbig",
    "helmClub": "SCZ",
    "helmClubCode": "B.122",
    "place": 2,
    "points": 126.58,
  },
  Object {
    "crew": "Lia-Marie Sturm",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Marie Rammin",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 3,
    "points": 123.16,
  },
  Object {
    "crew": "Kim Maje Petersen",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Malte Rathjens",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 4,
    "points": 119.74,
  },
  Object {
    "crew": "Aurel Köhler",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Franek Bladt",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 5,
    "points": 116.32,
  },
  Object {
    "crew": "Nora Josephine Wegner-Repke",
    "crewClub": "SVAG",
    "crewClubCode": "MV.042",
    "helm": "Frances Preisinger",
    "helmClub": "SVAG",
    "helmClubCode": "MV.042",
    "place": 6,
    "points": 112.89,
  },
  Object {
    "crew": "Johanna Stroh",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Richard Huth",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 7,
    "points": 109.47,
  },
  Object {
    "crew": "Franz Erpenbeck",
    "crewClub": "SCZ",
    "crewClubCode": "B.122",
    "helm": "Markus Müller",
    "helmClub": "SCZ",
    "helmClubCode": "B.122",
    "place": 9,
    "points": 102.63,
  },
  Object {
    "crew": "Fabian Helfrich",
    "crewClub": "WSVLS",
    "crewClubCode": "SC.031",
    "helm": "Bastian Klengel",
    "helmClub": "BSC",
    "helmClubCode": "SC.005",
    "place": 10,
    "points": 99.21,
  },
  Object {
    "crew": "Maximilian Scholle",
    "crewClub": "SVAG",
    "crewClubCode": "MV.042",
    "helm": "Johannes Wegner-Repke",
    "helmClub": "SVAG",
    "helmClubCode": "MV.042",
    "place": 11,
    "points": 95.79,
  },
  Object {
    "crew": "Analena Brusch",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Frauke Huth",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 12,
    "points": 92.37,
  },
  Object {
    "crew": "Lisa Marie Schwager",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Hanna Weber",
    "helmClub": "RSV",
    "helmClubCode": "SH.056",
    "place": 13,
    "points": 88.95,
  },
  Object {
    "crew": "Noah Never",
    "crewClub": "MYCR",
    "crewClubCode": "MV.020",
    "helm": "Christian Zils",
    "helmClub": "MYCR",
    "helmClubCode": "MV.020",
    "place": 14,
    "points": 85.53,
  },
  Object {
    "crew": "Maxi Zielinski",
    "crewClub": "SVAG",
    "crewClubCode": "MV.042",
    "helm": "Anna Chudzik",
    "helmClub": "SVAG",
    "helmClubCode": "MV.042",
    "place": 15,
    "points": 82.11,
  },
  Object {
    "crew": "Lea Vallentin",
    "crewClub": "SCP",
    "crewClubCode": "BG.012",
    "helm": "Gloria Szwajkowski",
    "helmClub": "SCP",
    "helmClubCode": "BG.012",
    "place": 16,
    "points": 78.68,
  },
  Object {
    "crew": "Karina Winter",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Aenne Bladt",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 17,
    "points": 75.26,
  },
  Object {
    "crew": "Thessa Müller",
    "crewClub": "SCP",
    "crewClubCode": "BG.012",
    "helm": "Sophia Böhme",
    "helmClub": "SCP",
    "helmClubCode": "BG.012",
    "place": 18,
    "points": 71.84,
  },
  Object {
    "crew": "Paula Enter",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Charlotte Knuth",
    "helmClub": "SCR",
    "helmClubCode": "MV.047",
    "place": 19,
    "points": 68.42,
  },
  Object {
    "crew": "Muriel Urbanski",
    "crewClub": "SVEG",
    "crewClubCode": "MV.015",
    "helm": "Luisa Voß",
    "helmClub": "SVEG",
    "helmClubCode": "MV.015",
    "place": 20,
    "points": 65,
  },
  Object {
    "crew": "Annalisa Tast",
    "crewClub": "SVEGS",
    "crewClubCode": "MV.015",
    "helm": "Julius Büchler",
    "helmClub": "SVEG",
    "helmClubCode": "MV.015",
    "place": 21,
    "points": 61.58,
  },
  Object {
    "crew": "Julia Peters",
    "crewClub": "SVEG",
    "crewClubCode": "MV.015",
    "helm": "Frederic Reiter",
    "helmClub": "SVEG",
    "helmClubCode": "MV.015",
    "place": 22,
    "points": 58.16,
  },
  Object {
    "crew": "Etienne Baschin",
    "crewClub": "VSW",
    "crewClubCode": "B.113",
    "helm": "Christian Müller",
    "helmClub": "SCZ",
    "helmClubCode": "B.122",
    "place": 23,
    "points": 54.74,
  },
  Object {
    "crew": "Lucas Maaß",
    "crewClub": "BSC",
    "crewClubCode": "SC.005",
    "helm": "Tom Jähnert",
    "helmClub": "BSC",
    "helmClubCode": "SC.005",
    "place": 24,
    "points": 51.32,
  },
  Object {
    "crew": "Margarethe Görs",
    "crewClub": "MYCR",
    "crewClubCode": "MV.020",
    "helm": "Johannes Schörck",
    "helmClub": "MYCR",
    "helmClubCode": "MV.020",
    "place": 26,
    "points": 44.47,
  },
  Object {
    "crew": "Anton Schmitter",
    "crewClub": "MYCR",
    "crewClubCode": "MV.020",
    "helm": "Viktoria Blindow",
    "helmClub": "ASVzR",
    "helmClubCode": "MV.035",
    "place": 27,
    "points": 41.05,
  },
  Object {
    "crew": "Marla Gröschner",
    "crewClub": "RSC92",
    "crewClubCode": "MV.055",
    "helm": "Max Cornelius",
    "helmClub": "RSC 92",
    "helmClubCode": "MV.055",
    "place": 28,
    "points": 37.63,
  },
  Object {
    "crew": "Jette Zielinski",
    "crewClub": "SVAG",
    "crewClubCode": "MV.042",
    "helm": "Carl Warning",
    "helmClub": "SVAG",
    "helmClubCode": "MV.042",
    "place": 29,
    "points": 34.21,
  },
  Object {
    "crew": "Sophie-Charlotte Gräfe",
    "crewClub": "RSC 92",
    "crewClubCode": "MV.055",
    "helm": "Marie-Luise Gräfe",
    "helmClub": "RSC 92",
    "helmClubCode": "MV.055",
    "place": 30,
    "points": 30.79,
  },
  Object {
    "crew": "Lucas Grapentin",
    "crewClub": "SVAG",
    "crewClubCode": "MV.042",
    "helm": "Joel Fabio Preisinger",
    "helmClub": "SVAG",
    "helmClubCode": "MV.042",
    "place": 31,
    "points": 27.37,
  },
  Object {
    "crew": "Paul Westphal",
    "crewClub": "ASVzR",
    "crewClubCode": "MV.035",
    "helm": "Oskar Thierbach",
    "helmClub": "MYCR",
    "helmClubCode": "MV.020",
    "place": 32,
    "points": 23.95,
  },
  Object {
    "crew": "Matthias Henning",
    "crewClub": "RSN",
    "crewClubCode": "BG.058",
    "helm": "Julian Mathwig",
    "helmClub": "RSN",
    "helmClubCode": "BG.058",
    "place": 33,
    "points": 20.53,
  },
  Object {
    "crew": "Fynn Sommer",
    "crewClub": "SCR",
    "crewClubCode": "MV.047",
    "helm": "Karl Albrecht Pulsack",
    "helmClub": "SCP",
    "helmClubCode": "BG.012",
    "place": 35,
    "points": 13.68,
  },
  Object {
    "crew": "Wilhelm Steinhäuser",
    "crewClub": "SCZ",
    "crewClubCode": "B.122",
    "helm": "Karl Steinhäuser",
    "helmClub": "SCZ",
    "helmClubCode": "B.122",
    "place": 36,
    "points": 10.26,
  },
  Object {
    "crew": "Kevin Gottschalk",
    "crewClub": "RSN",
    "crewClubCode": "BG.058",
    "helm": "Marc Dörfel",
    "helmClub": "RSN",
    "helmClubCode": "BG.058",
    "place": 37,
    "points": 6.84,
  },
  Object {
    "crew": "Lukas Bormann",
    "crewClub": "RSN",
    "crewClubCode": "BG.058",
    "helm": "Julian Kroll",
    "helmClub": "RSN",
    "helmClubCode": "BG.058",
    "place": 38,
    "points": 3.42,
  },
]
`;
